README.md

```diff
@@ -21,9 +21,9 @@ Install the [OpenTelemetry Collector Builder](https://github.com/open-telemetry/
 ### Get the code
 1. Clone the [telemetry generator repo](https://github.com/lightstep/telemetry-generator) to a directory of your choosing:
    1.  `$ cd ~/Code` (or wherever)
-   1.  `$ git clone https://github.com/lightstep/telemetry-generator`
-   1.  `$ cd telemetry-generator`
-1. Copy `hipster_shop.yaml` to `dev.yaml` for local development. Not strictly necessary (you can point the `TOPO_FILE` environment variable to any config file) but will potentially save heartache and hassle 😅 This file is in .gitignore, so it won't be included in your commits. If you want to share config changes, add them to a new example config file.
+   2. `$ git clone https://github.com/lightstep/telemetry-generator`
```

Comment:
just so you know, this change probably isn't necessary, lists with all value of `1.` should render with actual numbers. But I don't mind the change. It makes the raw markdown text more readable

Reply:
TIL, thank you! I was trying to address all the low hanging fruit code warnings in this PR.